guides/source/active_record_querying.md

```diff
@@ -1608,46 +1681,44 @@ a single object (see [Retrieving a Single Object Section](#retrieving-a-single-o
 have to be at the end of the statement.
 
 There are some examples below. This guide won't cover all the possibilities, just a few as examples.
-When an Active Record method is called, the query is not immediately generated and sent to the database,
-this just happens when the data is actually needed. So each example below generates a single query.
+When an Active Record method is called, the query is not immediately generated and sent to the database.
+ The query is sent only when the data is actually needed. So each example below generates a single query.
 
 ### Retrieving filtered data from multiple tables
 
 ```ruby
-Person
-  .select('people.id, people.name, comments.text')
-  .joins(:comments)
-  .where('comments.created_at > ?', 1.week.ago)
+Customer
+  .select('customers.id, customers.last_name, reviews.body')
+  .joins(:reviews)
+  .where('reviews.created_at > ?', 1.week.ago)
 ```
 
 The result should be something like this:
 
 ```sql
-SELECT people.id, people.name, comments.text
-FROM people
-INNER JOIN comments
-  ON comments.person_id = people.id
-WHERE comments.created_at > '2015-01-01'
+SELECT customers.id, customers.last_name, reviews.body
+FROM customers 
+INNER JOIN reviews
+  ON reviews.customer_id = customers.id 
+WHERE (reviews.created_at > '2019-01-08')
 ```
 
 ### Retrieving specific data from multiple tables
 
 ```ruby
-Person
-  .select('people.id, people.name, companies.name')
-  .joins(:company)
-  .find_by('people.name' => 'John') # this should be the last
+Book.select('books.id, books.title, authors.first_name')
+  .joins(:author)
+  .find_by(title: 'Abstraction and Specification in Program Development')
 ```
 
 The above should generate:
 
 ```sql
-SELECT people.id, people.name, companies.name
-FROM people
-INNER JOIN companies
-  ON companies.person_id = people.id
-WHERE people.name = 'John'
-LIMIT 1
```

Comment:
not sure if this line was meant to be removed, but assuming no so adding it back in https://github.com/rails/rails/pull/40349